app/views/providers/confirm_multiple_delegated_functions/show.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= page_template page_title: t('.heading', date: @used_multiple_delegated_functions_on), template: :basic do %>
```

Comment:
the page template tag and the form_with tag will need to be the other way around - this is a recent change from about a week ago! basically the error summary is rendered by the page template tag, and so you need to be able to pass the form to the page template tag so that it links correctly. Currently the link in the error summary on this page doesn't work (it should move focus to the fieldset). You can look at other pages as an example or happy to pair tomorrow if you like

Reply:
Switched! Yes makes sense. Is the new commit all that was needed?